tests/test_climate.py

```
@@ -150,3 +152,300 @@ async def test_scenario_3(hass: HomeAssistant, entry: MockConfigEntry, climate:
     assert climate.pulse_width_modulation_enabled
     assert climate.pwm.last_duty_cycle_percentage == 53.62
     assert climate.pwm.duty_cycle == (643, 556)
+
+
+@pytest.mark.parametrize(*[
+    "domains, data, options, config",
+    [(
+            [(TEMPLATE_DOMAIN, 1)],
+            {
+                CONF_MODE: MODE_FAKE,
```

Ruff (0.15.1)

[error] `CONF_MODE` may be undefined, or defined from star imports

(F405)